packages/sprite/src/Sprite.ts

```diff
@@ -1,13 +1,15 @@
-import { Point, ObservablePoint, Rectangle } from '@pixi/math';
-import { sign } from '@pixi/utils';
-import { Texture } from '@pixi/core';
 import { BLEND_MODES } from '@pixi/constants';
-import { Container } from '@pixi/display';
+import { Shader, Texture, Filter, Renderer, IBaseTextureOptions, TextureSource } from '@pixi/core';
```

Comment:
My hands just want to separate it to real `import` and `import type` but we cant do that before new TS is released, also our linter doesnt like two import lines from same module